src/backend/gporca/data/dxl/minidump/Gist-AOCOTable-NonLossy-BitmapIndexPlan.mdp

```diff
@@ -282,65 +282,47 @@ Gather Motion 3:1  (slice1; segments: 3)  (cost=0.00..0.00 rows=1 width=4)
+                <dxl:Ident ColId="1" ColName="b" TypeMdid="0.603.1.0"/>
+                <dxl:ConstValue TypeMdid="0.603.1.0" Value="AAAAAAAACEAAAAAAAAAQQAAAAAAAAPA/AAAAAAAAAEA="/>
+              </dxl:Comparison>
+            </dxl:RecheckCond>
+            <dxl:BitmapIndexProbe>
+              <dxl:IndexCondList>
                 <dxl:Comparison ComparisonOperator="~=" OperatorMdid="0.499.1.0">
                   <dxl:Ident ColId="1" ColName="b" TypeMdid="0.603.1.0"/>
                   <dxl:ConstValue TypeMdid="0.603.1.0" Value="AAAAAAAACEAAAAAAAAAQQAAAAAAAAPA/AAAAAAAAAEA="/>
                 </dxl:Comparison>
-              </dxl:RecheckCond>
-              <dxl:BitmapIndexProbe>
-                <dxl:IndexCondList>
-                  <dxl:Comparison ComparisonOperator="~=" OperatorMdid="0.499.1.0">
-                    <dxl:Ident ColId="1" ColName="b" TypeMdid="0.603.1.0"/>
-                    <dxl:ConstValue TypeMdid="0.603.1.0" Value="AAAAAAAACEAAAAAAAAAQQAAAAAAAAPA/AAAAAAAAAEA="/>
-                  </dxl:Comparison>
-                </dxl:IndexCondList>
-                <dxl:IndexDescriptor Mdid="0.566300.1.0" IndexName="boxindex"/>
-              </dxl:BitmapIndexProbe>
-              <dxl:TableDescriptor Mdid="0.566277.1.0" TableName="aoco_gist_tbl">
-                <dxl:Columns>
-                  <dxl:Column ColId="0" Attno="1" ColName="id" TypeMdid="0.23.1.0" ColWidth="4"/>
-                  <dxl:Column ColId="1" Attno="2" ColName="b" TypeMdid="0.603.1.0" ColWidth="32"/>
-                  <dxl:Column ColId="4" Attno="-1" ColName="ctid" TypeMdid="0.27.1.0" ColWidth="6"/>
-                  <dxl:Column ColId="5" Attno="-7" ColName="tableoid" TypeMdid="0.26.1.0" ColWidth="4"/>
-                  <dxl:Column ColId="6" Attno="-8" ColName="gp_segment_id" TypeMdid="0.23.1.0" ColWidth="4"/>
-                </dxl:Columns>
-              </dxl:TableDescriptor>
-            </dxl:BitmapTableScan>
-          </dxl:Sort>
-        </dxl:Result>
+              </dxl:IndexCondList>
+              <dxl:IndexDescriptor Mdid="0.566300.1.0" IndexName="boxindex"/>
+            </dxl:BitmapIndexProbe>
+            <dxl:TableDescriptor Mdid="0.566277.1.0" TableName="aoco_gist_tbl">
+              <dxl:Columns>
+                <dxl:Column ColId="0" Attno="1" ColName="id" TypeMdid="0.23.1.0" ColWidth="4"/>
+                <dxl:Column ColId="1" Attno="2" ColName="b" TypeMdid="0.603.1.0" ColWidth="32"/>
+                <dxl:Column ColId="4" Attno="-1" ColName="ctid" TypeMdid="0.27.1.0" ColWidth="6"/>
+                <dxl:Column ColId="5" Attno="-7" ColName="tableoid" TypeMdid="0.26.1.0" ColWidth="4"/>
+                <dxl:Column ColId="6" Attno="-8" ColName="gp_segment_id" TypeMdid="0.23.1.0" ColWidth="4"/>
+              </dxl:Columns>
+            </dxl:TableDescriptor>
+          </dxl:BitmapTableScan>
+        </dxl:Sort>
       </dxl:GatherMotion>
     </dxl:Plan>
   </dxl:Thread>
```

Comment:
please, fix query plan in plain text format in the comment section: remove `Result` node and refine cost values

Reply:
Thanks

Reply:
I should say other tests has different plan too, but their plan in comment section was not touched, because the difference was not caused by current patch.
As told on the one of our voice sessions, the problem of not actual mdp files is global and also related to their bodies.